python/whylogs/api/logger/segment_processing.py

```diff
@@ -57,7 +66,17 @@ def _process_simple_partition(
         # simple means we can segment on column values
         grouped_data = pandas.groupby(columns)
```

Comment:
Does this line work in multi-column segmentation where one column has a nan value and the other is non-nan?

Reply:
yes, but, for example a value (col1,col2) = (42, None) will get mapped to a group (42, float.nan), so this breaks when we try to do `get_group`, because float.nan is not equal to None. So the idea here is to avoid using get_group by checking if the value in a group is a float.nan and then applying the mask manually with `pandas[col].isna()`

Reply:
Thanks for the clarification!